akvo/rsr/spa/app/modules/editor/section7/recipient-regions/recipient-regions.jsx

```diff
@@ -114,6 +115,7 @@ const RecipientRegions = ({ formPush, validations, showRequired, errors }) => {
           </Button>
         )}
       />
+      <CalcTotalChecker section="7" path="recipientRegions" prop="percentage" />
```

Comment:
If there are recipient regions and countries both specified, the combined total should add up to 100%, not separately. 

See this [Zulip thread](https://akvo.zulipchat.com/#narrow/stream/131441-rsr/topic/Edit.20Error.20message.20IATI.20publication.20country.20percentage/near/173780667), for more information.